### ISSUE TITLE
Update copay rates page with information provided in new PDF

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#52621]
## Issue Description

A VA partner informed us that our [VA health care copay rates](https://www.va.gov/health-care/copay-rates/) page has information gaps, specifically related to Priority Group 5 copay requirements. They provided us with a PDF with info they're asking to add to the page.

PDF: https://www.va.gov/healthbenefits/resources/publications/IB10-431_copayment_requirements_by_priority_group.pdf

Per Danielle, the first step would be to review the page compared to the PDF for information gaps.

We'll open a separate ticket to complete the accessibility work needed to remove inaccessible tables from the page. When we do that, we'll reassess the page and consider a simplified “yes/no” summary version in addition to the specific rates as proposed (not necessarily in table form).  

We can add to the tasks once we've assessed the page and determined next steps.

## Tasks
- [ ] Set up call with Danielle to review work
- [ ] 
## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by aliyahblackmore** [comment]
Hi @laurwill - this is a word document with notes and questions from my review of the copay rates PDF and the rates page. 

[Notes on Copay rates_AB_2.21.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/10797771/Notes.on.Copay.rates_AB_2.21.docx)

**Comment by laurwill** [comment]
Hi @aliyahblackmore ,
Thanks for these notes! I'm adding more specific responses and next steps to your document, but first I want to confirm with @DanielleThierryUSDSVA:
- The live page lists copay requirements by priority group in some sections, and by disability rating in the outpatient and inpatient sections. Since disability ratings don't always map neatly to priority groups, I think it would be more accurate to list all copay rules by priority group number and leave disability ratings out of it. Do you agree?
- There are several discrepancies between the PDF and the live page. So before we start editing the Drupal, just want to confirm: Is the PDF attached here the source of truth for copay rate categories? I think we said it was, but it dates from May 2019 so I want to double check.
Thank you!

**Comment by laurwill** [comment]
Updates: Danielle confirmed we can list by priority group. We'll draft updates in Word and share with SMEs before moving into Drupal.

**Comment by laurwill** [comment]
Hi @DanielleThierryUSDSVA ,
We'd like to confirm best source for copay requirements by priority group before we suggest content updates. 
VA partner shared a [2019 PDF source](https://www.va.gov/healthbenefits/resources/publications/IB10-431_copayment_requirements_by_priority_group.pdf), but this [2021 PDF](https://www.va.gov/healthbenefits/resources/publications/IB_10-430_Copay_Rates_2021_Dec.pdf) has different information. Our live page aligns more closely with the 2021 PDF. So it's possible that the 2019 PDF they sent is outdated. 

Here are 2 examples of the discrepancies to note for them:
- For urgent care, 2021 source aligns with our live page. It separates the requirements by groups 1 to 5, group 6, and groups 7 to 8. But the 2019 source separates the requirements by groups 1 to 6 (applying the special authority rule to all 6 groups), and then groups 7 to 8. 
- For medication, 2021 source aligns with our live page. It separates requirements by groups 1 (no copays), and groups 2 to 8 (copays). But the 2019 source says that groups 1 and 4 have no copays. 

Thank you!

**Comment by RLHecht** [comment]
Moving to blocked until we figure out which source material we should look at.

**Comment by aliyahblackmore** [comment]
@strelichl based on the intake, it looks like we can close out this ticket. Is that accurate?
